applications/GeoMechanicsApplication/python_scripts/apply_vector_constraint_table_process.py

```diff
@@ -1,67 +1,82 @@
 import KratosMultiphysics
-import KratosMultiphysics.GeoMechanicsApplication as KratosGeo
+from KratosMultiphysics.GeoMechanicsApplication import ApplyComponentTableProcess
+from KratosMultiphysics import ApplyConstantScalarValueProcess
 
 def Factory(settings, Model):
     if(type(settings) != KratosMultiphysics.Parameters):
         raise Exception("expected input shall be a Parameters object, encapsulating a json string")
     return ApplyVectorConstraintTableProcess(Model, settings["Parameters"])
 
+
+def ComponentToIndex(component):
```

Comment:
Why is this a separate function? Is it only relevant inside the class or also outside? @classmethod or private member function seems more fitting.

Reply:
A class method probably makes more sense here. Changed according to your suggestion.